cfp_crawler/proposal/spiders/crawler.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import json
+from scrapy import signals
+
+
+class CrawlerSpider(scrapy.Spider):
+    name = 'crawler'
+    allowed_domains = ['in.pycon.org']
+    url = "https://in.pycon.org"
+    proposals = []
+    file = open("proposals.json", "w")
```

Comment:
The file is opened but never closed. You should close the file after use. The best way to do this would be to use `with`. Something like
```
with open("filename.json", "w") as f:
  f.write("data")
```
Move it inside the `spider_closed` method and if you want you make the filename as a member.

Reply:
oops! forgot to close it.

Reply:
Done